libs/market-status/.vscode/launch.json

```diff
@@ -0,0 +1,24 @@
+{
+	// Use IntelliSense to learn about possible attributes.
+	// Hover to view descriptions of existing attributes.
+	// For more information, visit: https://go.microsoft.com/fwlink/?linkid=830387
+	"version": "0.2.0",
+	"configurations": [
+	{
+		"type": "pwa-node",
+		"name": "vscode-jest-tests",
+		"request": "launch",
+		"program": "${workspaceFolder}/../../node_modules/jest/bin/jest",
+		"args": [
+			"--runInBand",
+			"--config",
+			"jest.config.js"
+		],
+    "envFile": "/TheCoinEnvironments/prod.env",
```

Comment:
When testing market-status, we want the prod variables so it can do the live fetch